src/DeviceInput/InputDevices/webDeviceInputSystem.ts

```diff
@@ -36,7 +36,7 @@ export class WebDeviceInputSystem implements IDeviceInputSystem {
     }
 
     public onDeviceDisconnected: (deviceType: DeviceType, deviceSlot: number) => void;
-    public onInputChanged: (deviceEvent: IDeviceEvent) => void;
+    public onInputChanged: (eventData: any) => void;

```

Comment:
Why `any` here?

Reply:
This isn't supposed to be any; changed to IUIEvent